test/SlowTests/Issues/RavenDB_23999.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FastTests;
+using Newtonsoft.Json;
+using Raven.Server.ServerWide;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_23999 : ClusterTestBase
+{
+    public RavenDB_23999(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [RavenFact(RavenTestCategory.ClientApi)]
+    public async Task Proxy_Will_Respect_Accept_Encoding_Header()
+    {
+        var result = await CreateRaftCluster(2);
```

Comment:
In 7.0 branch you should set zstd here for the requests compression (even though that's the current default), right? So it will be clear in the test that it's about the scenario with the usage of different compression methods.

Reply:
Spoke with Arek, all good here. There is no config option for that, just env variable.